community/blog/2022-06 .NET 6 finally comes to Arcus Templates in v0.8 release/README.md

```diff
@@ -0,0 +1,112 @@
+
+## Remove exception details from health report in Web API project template
+The Swagger OpenAPI documentation is by default available in the Web API project template (but can be excluded if needed). Adding Swagger to your API also means that the API models used during request/response will be externalized in the Swagger endpoint JSON. It was by inspecting this Swagger result that we saw a problem with the default application health endpoint that is available in the Web API project template. This endpoint used Microsoft's `HealthReport` model which can include exception details when the application is unhealthy. Generating the Swagger document caused failures because the exception details could contain possible internal type information which resulted in an enormous amount of types that needed to be included in the document. Also from a security perspective, this is not a good approach because it leaks information about the internal system. It could even leak application types that aren't used in the API.
+
+We solved this problem by adding a custom API model to the template. This custom model is an exact copy of Microsoft's `HealthReport` except that it doesn't contain any exception details. It still has a description, data, and tags to inform the user why the application is healthy or not, but it doesn't burden the recipient with internal information.
+
+The health endpoint available in the Web API project template uses this new API model as a way to expose the application's health. For more information on the Web API project template, see [our official documentation](https://templates.arcus-azure.net/features/web-api-template).
+
+## Hide JSON serialization behind maintainable extensions
+The Arcus templates are especially useful for speeding up development and removing the need to write boilerplate code. Message handling, secret management, we have all sorts of ways to abstract a problem into business components. Sometimes, though, it is not such a big piece of boilerplate functionality. The Web API project template used to have some lines of code to restrict and configure the JSON serialization. It was not a lot but it was enough to make the application startup code dirty. Not only dirty; but less maintainable. The code was removing output formatters, changing how the JSON serialization handles enumerations, etc. If we would decide to change this strategy, we couldn't provide support to consumers who already ran the template in their project.
+
+A previous Web API release made sure that our JSON serialization strategy is accessible through the MVC options, which makes it not only cleaner but also more maintainable.
```

Comment:
I don't completely understand this line, does this mean this was already solved in a previous release? Why would we mention it in the v0.8 release then?

Reply:
Well, it was the Web API release that made these extensions available, so it was solved in that release, but wasn't used until now.

Reply:
Ah I understand, thanks for the clarification